packages/cloud_firestore/android/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        api 'com.google.firebase:firebase-firestore:[11.4.0,12.0['
-        api 'com.google.firebase:firebase-core:[11.4.0,12.0['
+        api 'com.google.firebase:firebase-firestore:[11.6.0,12.0['
+        api 'com.google.firebase:firebase-core:[11.6.0,12.0['
```

Comment:
To any reviewers, please see the Firebase [Release Notes](https://firebase.google.com/support/release-notes/android) for 11.6.

Key quote: "Added support for deserializing field types with wildcard generic parameters (e.g. kotlin.Map)."